PWGCF/FemtoUniverse/Tasks/femtoUniversePairTaskTrackTrackMultKtExtended.cxx

```diff
@@ -321,29 +316,27 @@ struct femtoUniversePairTaskTrackTrackMultKtExtended {
     // ConfTOFPtMin - momentum value when we start using TOF; set to 1000 if TOF not needed
     // ConfNsigmaTPC -> TPC Sigma for momentum < ConfTOFPtMin
     // ConfNsigmaCombined -> TPC and TOF Sigma (combined) for momentum > ConfTOFPtMin
-    if (true) {
-      if (mom < twotracksconfigs.ConfTOFPtMinDe) {
-        if (TMath::Abs(nsigmaTPCDe) < twotracksconfigs.ConfNsigmaTPCDe) {
-          return true;
-        } else {
-          return false;
-        }
+
+    if (mom < twotracksconfigs.ConfTOFPtMinDe) {
+      if (TMath::Abs(nsigmaTPCDe) < twotracksconfigs.ConfNsigmaTPCDe) {
+        return true;
       } else {
-        if (TMath::Hypot(nsigmaTOFDe, nsigmaTPCDe) < twotracksconfigs.ConfNsigmaCombinedDe) {
-          return true;
-        } else {
-          return false;
-        }
+        return false;
+      }
+    } else {
+      if (TMath::Abs(nsigmaTOFDe) < twotracksconfigs.ConfNsigmaTOFDe && (TMath::Abs(nsigmaTPCDe) < twotracksconfigs.ConfNsigmaTPCDe)) {
+        return true;
+      } else {
+        return false;
       }
     }
-    return false;
   }
 
   /// Linear cut for clearer TPC Deuteron Sigma
   bool IsDeuteronNSigmaLinearCut(float mom, float nsigmaTPCDe, float nsigmaTOFDe, float tpcSignal)
   {
     if (lincut.ConfIsLine == true) {
-      if (mom > 0.8 && mom < 1.4) {
+      if (mom > 0 && mom < 1.4) {
```

Comment:
Maybe you want 0 and 1.4 to be configurables? Then, next time, you could update them in JSON and have no need to do PRs.

Reply:
Right, I will change them to configurables